File: src/main/java/xyz/citywide/next/lang/PlayerLanguageChangeEvent.java
package xyz.citywide.next.lang;

import net.minestom.server.entity.Player;
import net.minestom.server.event.trait.PlayerEvent;
import org.jetbrains.annotations.NotNull;

import java.util.Locale;

public record PlayerLanguageChangeEvent(Player player, Locale locale, boolean isClient) implements PlayerEvent {
    @Deprecated
    @Override
    public @NotNull Player getPlayer() {
        return player;
    }
}

File: src/main/kotlin/org/hypejet/hype/lang/PlayerLanguageChangeEvent.kt
package org.hypejet.hype.lang

import net.minestom.server.entity.Player
import net.minestom.server.event.trait.PlayerEvent
import org.jetbrains.annotations.NotNull
import java.util.*

data class PlayerLanguageChangeEvent(var pllayer: Player, val locale: Locale, val isClient: Boolean) :
    PlayerEvent {
    @Deprecated("")
    override fun getPlayer(): Player {
        return pllayer
    }
}